test/object_store/run

```diff
@@ -73,15 +73,17 @@ replication_opts = format_map({'class': 'NetworkTopologyStrategy',
 storage_opts = format_map({'type': 'S3',
                            'endpoint': s3_server_address,
                            'bucket': s3_public_bucket})
-conn.execute(("CREATE KEYSPACE test_ks WITH"
+ks = 'test_ks'
```

Comment:
I don't know if it's worth two extra lines. Tons of tests out there even written in C++ juse use, lieterally, `"ks.cf"`

Reply:
But I'm OK with committing this patch anyway, so it's up to you whether or not to remove one :shrug:

Reply:
> I don't know if it's worth two extra lines. Tons of tests out there even written in C++ juse use, lieterally, `"ks.cf"`

personally, i don't think this is the best practice. i am changing this because i happen to be staring at this piece of code long enough.

> But I'm OK with committing this patch anyway, so it's up to you whether or not to remove one shrug

thank you for indulging me. =)